umh.docs.umh.app/content/en/docs/production-guide/administration/normalize-kafka-topics.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Normalize Kafka Topics"
+content_type: task
+description: |
+    This page describes how to reduce the amount of Kafka Topics in order to
+    lower the overhead.
+weight: 50
+edition: community
+---
+
+<!-- overview -->
+
+Kafka excels at processing a high volume of messages but can encounter difficulties
+with excessive topics, which may lead to insufficient memory. The optimal Kafka
+setup involves minimal topics, utilizing the
+[event key](https://kafka.apache.org/intro#intro_concepts_and_terms) for logical
+data segregation.
+
+The diagram below illustrates the principle of combining multiple topics into a
+single topic with different keys, simplifying topic management.
+
+{{< mermaid >}}
+graph LR
+
+event1(Topic: umh.v1.acme.anytown.foo.bar<br>Value: 1)
+event2(Topic: umh.v1.acme.anytown.foo.baz<br>Value: 2)
+event3(Topic: umh.v1.acme.anytown<br>Value: 3)
+event4(umh.v1.acme.anytown.frob<br>Value: 4)
+
+event1 --> bridge
+event2 --> bridge
+event3 --> bridge
+event4 --> bridge
+bridge{{Topic merge point: 3}}
+
+subgraph Topic: umh.v1.acme
+gmsg1(Key: anytown.foo.bar<br>Value: 1)
+gmsg2(Key: anytown.foo.baz<br>Value: 2)
+gmsg3(Key: anytown<br>Value: 3)
+gmsg4(Key: anytown.frob<br>Value: 4)
+end
+
+bridge --> gmsg1
+bridge --> gmsg2
+bridge --> gmsg3
+bridge --> gmsg4
+
+{{< /mermaid >}}
+
+## {{% heading "prerequisites" %}}
+
```

Comment:
mention the default behavior of 1:1 MQTT / Kafka topics. And then say that a pre-requisite should be that you are experiencing challenges "out of memory" etc with the broker, which will start to appear after 100 different MQTT topics (there is a partition limit of 1000 partitions, and every topic in kafka has 6 partitions)